inputFiles/lagrangianContactMechanics/ContactMechanics_TFrac_base.xml

```diff
@@ -125,7 +125,8 @@
     <Coulomb
       name="fractureMaterial"
       cohesion="0.0"
-      frictionAngle="0.523598776"/>
+      frictionCoefficient="0.577350269"
+      apertureTableName="apertureTable"/>
```

Comment:
This should have a smoke test. I am guessing it doesn t since we didn't catch this error.

Reply:
Yes, its smoke test has been added to the integratedTests; but it has not been ran to be actively included: [add test for intersected frac problem #168](https://github.com/GEOSX/integratedTests/pull/168)

Reply:
Can you add it to the ats file in an integratedTests PR? Then, I will rebase and merge.

Reply:
It has been added to the` contactMechanics.ats ` in this [integratedTests PR], together with other uncovered smoke tests (https://github.com/GEOSX/integratedTests/pull/168)